app/src/main/java/org/simple/clinic/editpatient/EditPatientEffectHandler.kt

```diff
@@ -85,10 +87,12 @@ object EditPatientEffectHandler {
   private fun savePatientTransformer(
       patientRepository: PatientRepository,
       utcClock: UtcClock,
-      dateOfBirthFormatter: DateTimeFormatter
+      dateOfBirthFormatter: DateTimeFormatter,
+      schedulersProvider: SchedulersProvider
```

Comment:
Can't we pass the `scheduler` directly as a parameter instead of the entire `SchedulersProvider`?